railties/lib/rails/application/configuration.rb

```diff
@@ -170,20 +170,23 @@ def session_store(*args)
           case @session_store
           when :disabled
             nil
-          when :active_record_store
-            begin
-              ActionDispatch::Session::ActiveRecordStore
-            rescue NameError
-              raise "`ActiveRecord::SessionStore` is extracted out of Rails into a gem. " \
-                "Please add `activerecord-session_store` to your Gemfile to use it."
-            end
           when Symbol
             ActionDispatch::Session.const_get(@session_store.to_s.camelize)
           else
             @session_store
           end
         else
           @session_store = args.shift
+
+          if @session_store == :active_record_store
```

Comment:
Does that benefit outweigh a deprecation cycle? Seems a bit to quick in my opinion.

I don't quite understand your reasoning behind the move. I'm curious which tests were failing?